.github/workflows/spark.yaml

```
@@ -24,4 +26,33 @@ jobs:
       - name: package
         working-directory: clients/spark
         run: sbt package
+      - name: Install secret key for signing
+        if: github.event_name == 'workflow_dispatch'
+        run: |
+          cat <(echo -e '${{ secrets.OSSRH_GPG_SECRET_KEY }}') | gpg --batch --import
+          gpg --list-secret-keys --keyid-format LONG
+      - name: Prepare publish credentials
+        if: github.event_name == 'workflow_dispatch'
+        run: |
+          mkdir -p ~/.sbt
+          echo "realm=Amazon S3" > ~/.sbt/credentials
+          echo "host=treeverse-clients-us-east.s3.amazonaws.com" >> ~/.sbt/credentials
+          echo "user=${AWS_ACCESS_KEY_ID}"  >> ~/.sbt/credentials
+          echo "password=${AWS_SECRET_ACCESS_KEY}"  >> ~/.sbt/credentials
+          echo "realm=Sonatype Nexus Repository Manager" > ~/.sbt/sonatype_credentials
+          echo "host=s01.oss.sonatype.org" >> ~/.sbt/sonatype_credentials
+          echo "user=${OSSRH_USERNAME}" >> ~/.sbt/sonatype_credentials
+          echo "password=${OSSRH_TOKEN}" >> ~/.sbt/sonatype_credentials
```

I think this one should work without repeating the target file
```suggestion
          cat <<EOS > ~/.sbt/sonatype_credentials
          realm=Amazon S3
          host=treeverse-clients-us-east.s3.amazonaws.com
          user=${AWS_ACCESS_KEY_ID}
          password=${AWS_SECRET_ACCESS_KEY}
          realm=Sonatype Nexus Repository Manager
          host=s01.oss.sonatype.org
          user=${OSSRH_USERNAME}
          password=${OSSRH_TOKEN}
          EOS
```